### PR TITLE
Changed index page to categorize by platform use cases 

### PR DESCRIPTION
**Pull request by iremkatitas**
Added buttons for MCIN project/platform high-level categories for use cases mentioned in [Patrick's document](https://docs.google.com/document/d/1jIMG0UI26zHEExDFcd_UqbjH2H96oy1hIAzYg4yqqXc/edit):

- Data storage, management & sharing
- Data processing
- Data visualization
- Data governance policy, standards, best practice

- [ ]  Data governance category can be collapsed into broader category `Lowering the barriers to the practice of Open Science`

### CODE REVIEW

**Comment by christinerogers** [comment]
Hi @iremkatitas - i see this labelled waiting for review, could you also mark Whose review it's waiting for?  
e.g. Could you tag patrick, if it's not me, so we know? thanks

File: index.html
Comment (christinerogers) [inline index.html]: double the

File: index.html
Comment (christinerogers) [inline index.html]:
```suggestion
		at McGill University, <br> creates open neuroinformatics platforms to conduct computationally-intensive brain research using innovative mathematical and statistical approaches to integrate clinical, psychological or neuroimaging phenotypes with genotypic information. 
```
@iremkatitas @GHPBZ how do we feel about forcing line breaks?  there should also be one before "includes" so that CONP is printed on one line instead of split awkwardly.  
This definitely changes with browsers but i think these 2 line breaks will make sense generally.

**Review by christinerogers — CHANGES_REQUESTED** [review]
see comments inline: 
- 1 repetition
- requests for line breaks

Note we will probably end up with a footer containing logos across the bottom of the page.  probalby Ludmer and the Neuro and McGill.  Patrick is there 4th or more that wouldn't clutter too much?

File: index.html
Comment (iremkatitas) [reply index.html]: line breaks before "creates" and "including" added on latest commit

**Review by christinerogers — APPROVED** [review]

File: index.html
Comment (christinerogers) [reply index.html]: great, approving these changes  (and assuming you have looked to see it renders ok)

**Review by GHPBZ — APPROVED** [review]
Result looks good.  Thanks Irem.

**Pull request merged**
